R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -1079,19 +1079,19 @@ There are three main object classes in SparkR you may be working with.
     + `sdf` stores a reference to the corresponding Spark Dataset in the Spark JVM backend.
     + `env` saves the meta-information of the object such as `isCached`.
 
-It can be created by data import methods or by transforming an existing `SparkDataFrame`. We can manipulate `SparkDataFrame` by numerous data processing functions and feed that into machine learning algorithms.
+    It can be created by data import methods or by transforming an existing `SparkDataFrame`. We can manipulate `SparkDataFrame` by numerous data processing functions and feed that into machine learning algorithms.
```

Comment:
Minor points, but yes, the four spaces indicate that the text following should be aligned with the bullet point. Otherwise, it will start as a new paragraph and have the wrong indention.  
You will see the difference after compiling the Rmarkdown file.